src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -56,14 +56,20 @@
 #include "utils/inval.h"
 #include "utils/lsyscache.h"
 #include "utils/memutils.h"
+#if (PG_VERSION_NUM >= 100000)
+#include "utils/varlena.h"
+#endif
 
+#define LOCK_RELATION_IF_EXISTS "SELECT lock_relation_if_exists('%s', '%s');"
 
 static List * ModifyMultipleShardsTaskList(Query *query, List *shardIntervalList, TaskType
 										   taskType);
 static bool ShouldExecuteTruncateStmtSequential(TruncateStmt *command);
+static LOCKMODE LockModeTextToLockMode(const char *lockModeName);
 
 
 PG_FUNCTION_INFO_V1(master_modify_multiple_shards);
+PG_FUNCTION_INFO_V1(lock_relation_if_exists);
```

Comment:
we also include matching declaration file in header files. Candidates are `master_protocol.h` or `multi_utility.h`

Reply:
👍